ui/analyse/src/study/relay/relayView.ts

```diff
@@ -78,20 +78,20 @@ export function renderStreamerMenu(relay: RelayCtrl) {
   );
 }
 
-function renderBoardView(ctx: ViewContext) {
+function renderBoardView(ctx: RelayViewContext) {
   const { ctrl, deps, study, gaugeOn, relay } = ctx;
   // no worries about FOUC when overriding grid constraints here, it's all snab
   const cols = Number(window.getComputedStyle(document.body).getPropertyValue('--cols'));
```

Comment:
isn't this forcing the browser to do a complete page render, in order to get the computed styles, before snabbdom can start producing any vdom?

Reply:
I checked with the browser performance tools and it doesn't seem to be a bottleneck.